example/ionic-angular-v2/package.json

```diff
@@ -14,7 +14,7 @@
     "@ionic-native/splash-screen": "^5.0.0",
     "@ionic-native/status-bar": "^5.0.0",
     "@ionic/angular": "^5.0.0",
-    "@sentry/angular": "7.70.0-beta.1",
+    "@sentry/angular": "7.73.0",
```

Comment:
Not related to this PR, but should we use `@sentry/angular-ivy` since the `@angular` packages in the examples are `~13`?